internal/pkg/service/stream/storage/level/local/diskwriter/network/rpc/pb/networkFile.proto

```diff
@@ -8,6 +8,7 @@ import "google/protobuf/timestamp.proto";
 
 service NetworkFile {
   rpc Open(OpenRequest) returns (OpenResponse) {}
+  rpc WaitForServerTermination(WaitForServerTerminationRequest) returns (stream ServerIsTerminatingResponse);
   rpc Write(WriteRequest) returns (WriteResponse) {}
```

Comment:
gRPC has only one direction of calls, `client -> server`, `Source Node -> Disk Writer Node`.

But we need to signal `Disk Writer Node` termination to the `SourceNode` ( there is the pipeline start).

---------

So we use `stream`, the `client - SourceNode` opens an long-lived stream, and `server - Disk Writer Node` can send termination request.